frontend/apps/docs/content/docs/cli/ci-cd.mdx

```diff
@@ -1,6 +1,163 @@
+</Files>
+
+The `schema.json` file contains the parsed schema data in JSON format, which `index.html` reads to render the ER diagram.  
+
+Additional notes:
+
+- You can place these files under a subdirectory if needed.  
+- This output is similar to a typical Vite-built SPA. Refer to [Deploying a Static Site (Vite official documentation)](https://vite.dev/guide/static-deploy.html) for more details on deployment to common hosting providers.
+
+## Using the `init` Command
+
+The `init` command in the [Liam ERD CLI](/docs/cli) provides an interactive way to generate configuration files or GitHub Actions templates tailored to your project, automatically handling tasks like:
+
+- Specifying the input file (`--input`) and schema format (`--format`)
+- Generating sample GitHub Actions workflows
+
+If you’re new to Liam ERD or unsure about configurations, simply run:
+
+```bash
+npx @liam-hq/cli init
+```
+
+Follow the on-screen prompts to set up Liam ERD quickly.
+
+## Example: Prisma + GitHub Actions + Cloudflare Pages
+
+Below is a more advanced example using `schema.prisma` with GitHub Actions to generate and deploy ER diagrams to Cloudflare Pages.  
+Cloudflare Pages supports simple access restrictions via Cloudflare Access, enabling you to limit diagram visibility to internal team members only.
+
+```yaml
+name: Deploy ERD (Prisma) to Cloudflare Pages
+
+on:
+  push:
+    branches:
+      - main
+    # NOTE: If you only want to trigger this when the schema file is updated:
+    paths:
+      - prisma/schema.prisma
+
+jobs:
+  build-and-deploy-erd:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      deployments: write
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Generate ER Diagrams
+        run: npx @liam-hq/cli erd build --input prisma/schema.prisma --format prisma
+      - name: Deploy ERD to Cloudflare Pages
+        uses: cloudflare/wrangler-action@v3
+        with:
+          apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN_SAMPLE_PRISMA }}
+          accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID_SAMPLE_PRISMA }}
+          command: pages deploy ./dist --project-name=prisma-with-cloudflare-pages
+          gitHubToken: ${{ secrets.GITHUB_TOKEN }}
+```
+
+We plan to share more examples for other static hosting platforms. The above workflow is adapted from [Deploy ERD to Cloudflare Pages Sample](https://github.com/liam-hq/liam-erd-samples/blob/main/.github/workflows/prisma-with-cloudflare-pages.yml). To see more examples, visit [Liam ERD Samples](https://github.com/liam-hq/liam-erd-samples).
+
+## Common Pitfalls
+
+### Security and Privacy
+- If you are using a static hosting service like Cloudflare Pages, Vercel, or Netlify, **avoid exposing your private repository schema** by default.
+- Configure organizational authentication so that **only internal members** can view it.
+
+### Performance with Large Schemas
+- If your schema contains a large number of tables, initial load times can increase.
+
```

Comment:
This one doesn't seem to need explaining. What do you think?

Reply:
@MH4GF My concern was that it might become unstable when handling a very large number of tables (such as 1000+).  

However, since there's no concrete workaround at the moment, it might be fine to leave it out. I'll remove it!

Reply:
> However, since there's no concrete workaround at the moment, it might be fine to leave it out.

I think so too 💭 Let's discuss solutions in the future.